packages/fhir-converter/src/templates/cda/Resources/Condition.hbs

```diff
@@ -30,6 +32,12 @@
                 {{/each}}
             ],
         {{/if}}
+
+        "category":{{>DataType/CodeableConcept.hbs code=conditionEntry.entryRelationship.observation.code}},
```

Comment:
do we know `conditionEntry.entryRelationship.observation.code` will always be defined?

maybe we should put this behind an `if`?

Reply:
I noticed that when it's not present, the "category" attribute just doesn't get inserted. So the `if` block seems to be redundant